### DISCUSSION TITLE
Darwin access token

### DISCUSSION THREAD

**Discussion by alivingstone70** [jpsingleton/Huxley2#20] [General]
Not sure if I am missing something obvious.
I followed the link to register for a darwin token, but after completing this I see no place that has a guid there.
I see Darwin File Information (access key and token)
Darwin FTP Information
Darwin Topic Information
Is there somewhere else to get a token?

**Comment by jpsingleton** [comment]
I suspect you have signed up for the push port rather than the web service but it's been a while since I've had to do this.
This isn't a Huxley issue so perhaps post on https://groups.google.com/g/openraildata-talk with the detailed steps you took or some screenshots.

**Reply by alivingstone70** [reply]
Thats what I did.  Signed up for Darwin, but not the webservice.  All working now.
